library/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -634,24 +633,20 @@ internal class RumViewManagerScopeTest {
 
         val appStartTimeNs = forge.aLong(min = 0, max = fakeEvent.eventTime.nanoTime)
         whenever(mockAppStartTimeProvider.appStartTimeNs) doReturn appStartTimeNs
-        whenever(mockSdkCore.processImportance) doReturn RunningAppProcessInfo.IMPORTANCE_FOREGROUND
+        DdRumContentProvider.processImportance = RunningAppProcessInfo.IMPORTANCE_FOREGROUND
 
         // When
         testedScope.handleEvent(fakeEvent, mockWriter)
 
         // Then
-        val appStartTimestamp = TimeUnit.NANOSECONDS.toMillis(
-            (
-                TimeUnit.MILLISECONDS.toNanos(fakeEvent.eventTime.timestamp) -
-                    fakeEvent.eventTime.nanoTime
-                ) +
-                appStartTimeNs
-        )
+        val eventTimeNanos = TimeUnit.MILLISECONDS.toNanos(fakeEvent.eventTime.timestamp)
+        val timestampNs = (eventTimeNanos - fakeEvent.eventTime.nanoTime) + appStartTimeNs
+        val timestampMs = TimeUnit.NANOSECONDS.toMillis((timestampNs))
```

Comment:
format: redundant brackets

```suggestion
        val timestampMs = TimeUnit.NANOSECONDS.toMillis(timestampNs)
```